java/integTest/src/test/java/glide/TestConfiguration.java

```diff
@@ -1,18 +1,58 @@
-/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
 package glide;
 
 import com.vdurmont.semver4j.Semver;
+import glide.api.GlideClient;
+import glide.api.models.commands.InfoOptions;
+
 import java.util.Arrays;
 
+import static glide.TestUtilities.commonClientConfig;
+
 public final class TestConfiguration {
     // All servers are hosted on localhost
     public static final int[] STANDALONE_PORTS = getPortsFromProperty("test.server.standalone.ports");
     public static final int[] CLUSTER_PORTS = getPortsFromProperty("test.server.cluster.ports");
-    public static final Semver SERVER_VERSION = new Semver(System.getProperty("test.server.version"));
+    private static final GlideClient standaloneClient;
+    public static final Semver SERVER_VERSION;
+
+    static {
+        try {
+            standaloneClient = GlideClient.createClient(commonClientConfig().build()).get();
```

Comment:
```suggestion
            var standaloneClient = GlideClient.createClient(commonClientConfig().build()).get();
```
No need static lifetime for the client

Reply:
Updated.